### PR TITLE
Add variable for the ID of the account that owns the AMI

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

In this pull request I add an input variable for the ID of the account that owns the AMI.  The default value for this variable is `self`, so the module behaves just as it did previously unless you specify a different value for the new value.

## 💭 Motivation and Context

Previously the owner was hardcoded to `self`, which will not work when deploying to the COOL.  We need to specify the images account ID as the owner of the AMI in that case.

## 🧪 Testing

All the pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: ami.tf [hunk 1]
```diff
@@ -32,6 +25,8 @@ data "aws_ami" "openvpn" {
     values = ["ebs"]
   }
 
-  owners      = ["self"]
+  owners = [
```
Comment (felddy) [inline ami.tf:28]: I didn't realize this is an array.  This raises interesting questions / solutions.

**Review by felddy — APPROVED** [review]
Yup.  This is important and should be back ported to the skeleton.

**Review by dav3r — APPROVED** [review]
Yup yup. 👍

**Review by mcdonnnj — APPROVED** [review]
👍

**Pull request merged**